web/public/js/services/user-utils.js

```diff
@@ -105,5 +105,22 @@ angular.module('cpZenPlatform').factory('userUtils', ['$location', '$window', '$
     return  avatar || overallDefault;
   }
 
+  userUtils.getTitleForUserTypes = function (userTypes) {
+    var title = [];
+    if (_.includes(userTypes, 'attendee-u13')) {
+      title.push($translate.instant('Ninja (Under 13)'));
```

Comment:
Well, it's useful for Dojo admins to know if a kid is under or over 13 for things like whether a parent should be present. For any automatic updating of accounts to attendee-o13 (or semi-auto, like in this functionality – setting a youth as a mentor/champion will also update their under/over 13 status), we should probably fix CoderDojo/community-platform#1017 before pushing.